### ISSUE TITLE
typescript support? or typescript integration?

### ISSUE THREAD

**Issue by devdoomari3** [levjj/esverify#2]
I really like the concept, and thought it would be better to have a typescript-integration.

- most typescript-codes already have types have types

    ```typescript
    function max(a: number, b: number) {
      requires(typeof(a) === "number"); // not necessary
      requires(typeof(b) === "number"); // ...
      ensures(res => res >= a); 
    }
    ```

- typescript has its own parsers you can use
- typescript has a big collection of type-stubs for JS libraries (definitelytyped)

**Comment by devdoomari3** [comment]
sorry your readme already have typescript-as-input-language.

**Comment by levjj** [comment]
Yes, this would definitely be useful. Also, it should not be too difficult to implement. (contributions welcome!)
